R/interpret.R

```diff
@@ -53,6 +54,8 @@
 #'              1-D PDP plots, 2-D PDP plots, and grid points for building the distilled model.
 #'              This saves the uncentered calculations.
 #' @field ale.grid A list that caches the saved predictions for the ALE plots
+#' @field pdp.rank A string to select which PDP ranking methodology
```

Comment:
When there is a parameter for a function that takes string values, we should always list the potential options and describe the behavior for each case in the documentation. For example: "Should be one of c("Variance","CP.detect","CP.local"). When set to "Variance" ...., when set to "CP.detect" .... etc."

Reply:
We should also not include white space for potential options. So instead of using "Inflection Point" I would use "InflectionPoint" or "Inflection.Point"